### DISCUSSION TITLE
Lenovo legion tool kit WORKS GREAT with Legion Y520

### DISCUSSION THREAD

**Discussion by PedGit025** [BartoszCichecki/LenovoLegionToolkit#729] [Q&A] [answered]
Thanks for creating this tool

I wonder is there a way for me to remove the warning after turning on the PC that says my device is not compatible, i tried adding legion toolkit at startup but it didnt fix it.

english not my first language

**Comment by BartoszCichecki** [comment, marked as answer]
Check the FAQ and you will find what you seek :)
